client/src/app/account-card/account-card.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { AccountModel } from './account.model';
+
+@Component({
+  selector: 'app-account-card',
+  templateUrl: './account-card.component.html',
+  styleUrls: ['./account-card.component.scss'],
+})
+export class AccountCardComponent {
+  @Input() public account!: AccountModel;
```

Comment:
for what do use add '!' ?

Reply:
because it will be type of AccountModel at runtime, if i omit '!' it gives me error

Reply:
okey, let it be like this